modules/sdk-lib-mpc/src/tss/ecdsa-dkls/dkg.ts

```diff
@@ -111,9 +135,10 @@ export class Dkg {
     }
     if (this.dklsKeyShareRetrofitObject) {
       this.dkgSession = this.seed
-        ? KeygenSession.initKeyRotation(this.dklsKeyShareRetrofitObject, new Uint8Array(this.seed))
-        : KeygenSession.initKeyRotation(this.dklsKeyShareRetrofitObject);
+        ? this.getDklsWasm().KeygenSession.initKeyRotation(this.dklsKeyShareRetrofitObject, new Uint8Array(this.seed))
+        : this.getDklsWasm().KeygenSession.initKeyRotation(this.dklsKeyShareRetrofitObject);
     } else {
+      const { KeygenSession } = this.getDklsWasm();
```

Comment:
can we put this before the if/else block so it can be used within both if and else